crates/sui/src/client_commands.rs

```diff
@@ -2167,3 +2213,45 @@ impl Display for SwitchResponse {
         write!(f, "{}", writer)
     }
 }
+
+/// Request tokens from the Faucet for the given address
+pub async fn request_tokens_from_faucet(
+    address: SuiAddress,
+    url: String,
+) -> Result<(), anyhow::Error> {
+    let address_str = address.to_string();
+    let json_body = json![{
+        "FixedAmountRequest": {
+            "recipient": &address_str
+        }
+    }];
+
+    // make the request to the faucet JSON RPC API for coin
+    let client = reqwest::Client::new();
+    let resp = client
+        .post(&url)
+        .header("Content-Type", "application/json")
+        .json(&json_body)
+        .send()
+        .await?;
+    println!("Requested gas from faucet: {url}");
+    println!(
```

Comment:
What's our current stance on `println!` vs `tracing` vs other output methods for the CLI? Do we have a standard now?

Reply:
I've been thinking on how to do this. What we wanted to do is to use `eprintln` for stderr stuff, and the rest should be mostly tabulated output. What I wanted to have as output for this is the new coin object id, but that's difficult because of the rate limiter on the faucet server (in my tests, I run the request and then the next request to check for task status returned a 429, so not easy). An alternative is to call the gas command before the faucet, and then in a loop wait until we have 1 extra coin, and then do a diff to get the id. It's hacky, but this solves the rate limiter issue. 

Thoughts?